src/js/components/Diagram/Diagram.js

```diff
@@ -222,20 +273,49 @@ const Diagram = forwardRef(({ connections, ...rest }, ref) => {
             colorName = colors[index % colors.length];
           }
 
+          let arrowMarker = null;
```

Comment:
This is a smaller comment but to be more flexible for future, should we name this variable `endpoint`? If needed we can alias the prop as `endpointProp`.

Reply:
Since, `endpoint` is already a prop we are adding in Diagram component, naming the `arrowMarker` variable to `endpointProp` will create a confusion as it's expected type to be `'arrow' | { from?: 'arrow'; to?: 'arrow' }`. How about if we name this variable as `endpointMarker` ?
The reason, I have choosed this name - `endpointMarker` variable is because we are storing the `marker` tag with the openArrow in it and in future I believe for other `endpoints` we will be using `marker` for it. Please, let me know your thoughts